openai_connector/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Openai Connector",
+    "version": "15.0.1.0.0",
+    "author": "Quatile",
+    "license": "AGPL-3",
+    "website": "https://www.quartile.co",
+    "depends": ["base_setup", "web"],
+    "data": [
+        "security/ir.model.access.csv",
+        "views/res_config_settings_views.xml",
+        "views/openai_vision_session.xml",
```

Comment:
To stick the practice.
```suggestion
        "views/openai_vision_session_views.xml",
```

Reply:
Done